### PR TITLE
London Class 8 - Hatice Aydogan - CYF-Hotel-React - Week 2

### PR DESCRIPTION
**Pull request by Hatice06**

### CODE REVIEW

File: src/Bookings.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import React from "react";
+import React, { useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(FakeBookings);
```
Comment (ewintram) [inline src/Bookings.js:7]: well done for setting the initial state using fakeBookings.json. Remember only React components should begin with a capital letter, so line 4 should be
```
import fakeBookings from "./data/fakeBookings.json";
```

File: src/Order.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = props => {
+  const [orders, setOrders] = useState(0);
+
+  const orderOne = () => {
+    setOrders(orders + 1);
```
Comment (ewintram) [inline src/Order.js:8]: well done for using the setOrders state hook!

File: src/RestaurantButton.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RestaurantButton = props => {
+  return (
+    <button className="btn btn-primary" onClick={props.orderOne}>
```
Comment (ewintram) [inline src/RestaurantButton.js:5]: well done for passing the orderOne function as a prop

File: src/SearchResults.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    </table>
+  );
+};
+
+const TableRow = props => {
+  const [selected, setSelected] = useState(false);
+
+  const clickHandler = () => {
+    selected === false ? setSelected(true) : setSelected(false);
+  };
```
Comment (ewintram) [inline src/SearchResults.js:34]: Good use of a ternary! You could shorten this to 
```
const clickHandler = () => {
    setSelected(!selected);
};
```

File: src/SearchResults.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <tr onClick={clickHandler} className={selected ? "selected" : null}>
```
Comment (ewintram) [inline src/SearchResults.js:36]: well done using the state to determine the className

File: src/SearchResults.js (hunk 4 above)
Comment (Hatice06) [reply src/SearchResults.js:34]: Thanks Ellie, makes more sense , I changed it.
